### PR TITLE
PMM-7 Lint fix.

### PR DESCRIPTION
**Pull request by JiriCtvrtka**

### CODE REVIEW

File: managed/utils/interceptors/interceptors.go [hunk 1]
```diff
@@ -75,8 +76,7 @@ func logRequest(l *logrus.Entry, prefix string, f func() error) (err error) {
 		}
 	}()
 
-	err = f()
-	return //nolint:nakedret
+	return f()
```
Comment (BupycHuk) [inline managed/utils/interceptors/interceptors.go:79]: we still need `err = f()` here for logic in `defer`

Comment (BupycHuk) [reply managed/utils/interceptors/interceptors.go:79]: please check other places where you removed `err = .*` to make sure that `err` isn't used in `defer` function

Comment (JiriCtvrtka) [reply managed/utils/interceptors/interceptors.go:79]: Yeah right. Anyway PR is not ready yet.

**Comment by JiriCtvrtka** [comment]
Fix for nonamedreturns and nakedret lints. Right now I am skipping it in interceptors files since it is creating bug with EOF instead on "internal server error".

**Review by BupycHuk — APPROVED** [review]

**Review by ademidoff — APPROVED** [review]

**Pull request merged**